doc/develop/getting_started/index.rst

```diff
@@ -65,7 +65,7 @@ The current minimum required version for the main dependencies are:
      - 3.20.0
 
    * - `Python <https://www.python.org/>`_
-     - 3.6
+     - 3.8
```

Comment:
Sorry, but I disagree with this. Zephyr doesn't require Python 3.6, except for Python support in GDB I assume. But that's not enough to force 3.8 on everyone. That said, I am OK with upgrading the minimum required Python from 3.6 to 3.8 or higher, but let's discuss this in a separate PR where we see the benefits of going as high as possible in one swoop.

Reply:
Fine dropping this. Reason for the change was the fact that our default GDB used to require python, but pending confirmation that is no longer the case, I agree this change should be dropped.